conans/client/toolchain/cmake/templates/blocks/oshost/android.cmake

```diff
@@ -0,0 +1,3 @@
+set(CMAKE_ANDROID_ARCH_ABI {{ tc.android_arch_abi }})
```

Comment:
right now the order of construction is not so obvious and implicit, e.g. I don't understand if `android.cmake` goes before or after `crossbuild.cmake`? the order might be important, as different parts may override each other.